accounts-journey/presentation/ui/accounts_list/AccountsListViewController.swift

```diff
@@ -57,22 +57,9 @@ final class AccountsListViewController: UIViewController {
         setupView()
         setupSearchController()
         setupBindings()
-    }
-    
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
-        
         viewModel.onEvent(.getAccounts)
     }
     
-    override func loadView() {
-        super.loadView()
-        view.addSubview(accountsListTableView)
-        view.addSubview(loadingView)
-        setupLayout()
-        
```

Comment:
this is what is written in the apple docs:

You should never call this method directly. The view controller calls this method when its [view](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621460-view) property is requested but is currently nil. This method loads or creates a view and assigns it to the [view](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621460-view) property.

Reply:
it also mentions about `assign the root view of that hierarchy to the view controller’s view property.` but honestly i do not see this happening, am i missing something?

Reply:
From the documentation [here](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview)

The documentation says the following about `loadView` 
- "You should never call this method directly".  This is not being done in the commented out code - First Paragraph
- "You can override this method in order to create your views manually", which is what we're doing in the section above. - Last paragraph. 

Which means that we can still `override func loadView` since we're programmatically creating / adding the views.

Reply:
Disregard my comments.